dataland-frontend/tests/e2e/specs/low/pages/TaxonomySearch.ts

```diff
@@ -0,0 +1,32 @@
+describe('EU Taxonomy Page', function () {
```

Comment:
was ist mit low, medium und high in unit tests Ordner gemeint? wieso braucht man das?